docs/marketplace/dev/glue-api-guides/202204.0/marketplace-shopping-lists/managing-marketplace-shopping-lists.md

```diff
@@ -353,7 +353,7 @@ To retrieve a shopping list, send the request:
 
 | QUERY PARAMETER | DESCRIPTION | EXEMPLARY VALUES |
 | --- | --- | --- |
-| include | Adds resource relationships to the request. | shopping-list-items, concrete-products|
+| include | Adds resource relationships to the request. | shopping-list-items, concrete-products, merchants, product-offers |
```

Comment:
Please write the exemplary values as a list. They are better readable in list than in one line